sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.bigconfig.yml

```diff
@@ -0,0 +1,20 @@
+type: BIGCONFIG_FILE
+table_deployments:
+- collection:
+    name: Test
+  deployments:
+  - fq_table_name: {{ project_id }}.{{ derived_dataset }}.{{ target_table }}
+    table_metrics:
+    - metric_type:
+        type: PREDEFINED
+        predefined_metric: FRESHNESS
+      metric_schedule:
+        named_schedule:
+          name: Default Schedule - 17:00 UTC
+    - metric_type:
+        type: PREDEFINED
+        predefined_metric: VOLUME
+      metric_schedule:
+        named_schedule:
+          name: Default Schedule - 17:00 UTC
```

Comment:
```suggestion
          name: Default Schedule - 13:00 UTC
```

Reply:
Do you mind changing this also here: https://github.com/mozilla/bigquery-etl/blob/d741403a9b17beef18a92767716dd2d84e6964b5/bigquery_etl/cli/monitoring.py#L176

and here: https://github.com/mozilla/bigquery-etl/blob/d741403a9b17beef18a92767716dd2d84e6964b5/sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/releases_v1/bigconfig.yml